.gitignore

```
@@ -360,4 +359,4 @@ MigrationBackup/
 .ionide/
 
```

The newline at the end of the file has been modified. It is generally a good practice to ensure a consistent trailing newline at the end of files for compatibility with POSIX systems and tools.

**Suggested change**
```suggestion
FodyWeavers.xsd

```